dist/auth.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
```

Comment:
I'd rather not commit the compiled files as they don't provide any additional information. Perhaps we could just keep a `dist/.gitkeep` file to prevent babel from crashing when the directory doesn't exist?